ConfigMappingsTest.generateToString expects properties to always be in the same order

[ConfigMappingsTest.generateToString](https://github.com/smallrye/smallrye-config/blob/82756a65b2dae87ed185c217695ffa7a666f04d3/implementation/src/test/java/io/smallrye/config/ConfigMappingsTest.java#L193) is checking the result of the generated `toString` method against a hardcoded result. However, the order in which the properties are output is determined by the order of the properties in `ConfigMappingInterface` which are derived from calling [Class.getDeclaredMethods](https://github.com/smallrye/smallrye-config/blob/82756a65b2dae87ed185c217695ffa7a666f04d3/implementation/src/main/java/io/smallrye/config/ConfigMappingInterface.java#L725) on the interface type. That method is not guaranteed to return the methods in any particular order and this can change between different JVMs.

The result is that the test can fail on some JVMs...

```
[INFO] Running io.smallrye.config.ConfigMappingsTest
[ERROR] Tests run: 9, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.010 s <<< FAILURE! -- in io.smallrye.config.ConfigMappingsTest
[ERROR] io.smallrye.config.ConfigMappingsTest.generateToString -- Time elapsed: 0.002 s <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <ToStringMapping{host=localhost, port=8080, aliases=[Alias{name=prod-1}, Alias{name=prod-2}]}> but was: <ToStringMapping{aliases=[Alias{name=prod-1}, Alias{name=prod-2}], host=localhost, port=8080}>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1142)
	at io.smallrye.config.ConfigMappingsTest.generateToString(ConfigMappingsTest.java:193)
```
expected:
`<ToStringMapping{host=localhost, port=8080, aliases=[Alias{name=prod-1}, Alias{name=prod-2}]}>`
but was:
`<ToStringMapping{aliases=[Alias{name=prod-1}, Alias{name=prod-2}], host=localhost, port=8080}>`

Same content but in a different order.

--- a/implementation/src/test/java/io/smallrye/config/ConfigMappingsTest.java
+++ b/implementation/src/test/java/io/smallrye/config/ConfigMappingsTest.java
@@ -190,8 +190,14 @@ public class ConfigMappingsTest {
                 .build();
 
         ToStringMapping mapping = config.getConfigMapping(ToStringMapping.class);
-        assertEquals("ToStringMapping{host=localhost, port=8080, aliases=[Alias{name=prod-1}, Alias{name=prod-2}]}",
-                mapping.toString());
+        String toString = mapping.toString();
+        assertTrue(toString.contains("ToStringMapping{"));
+        assertTrue(toString.contains("host=localhost"));
+        assertTrue(toString.contains("port=8080"));
+        assertTrue(toString.contains("port=8080"));
+        assertTrue(toString.contains("aliases=["));
+        assertTrue(toString.contains("Alias{name=prod-1}"));
+        assertTrue(toString.contains("Alias{name=prod-2}"));
     }
 
     @ConfigMapping(prefix = "server")

